src/main/java/com/Example/BankManagement/Controller/AccountController.java

```diff
@@ -0,0 +1,35 @@
+package com.Example.BankManagement.Controller;
+
+import com.Example.BankManagement.Entity.Account;
+import com.Example.BankManagement.Entity.Customer;
+import com.Example.BankManagement.Service.AccountService;
+import com.Example.BankManagement.response.AccountResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/accounts")
+public class AccountController {
+
+    @Autowired
+    private AccountService accountService;
+
+    @PostMapping("/details")
+    public ResponseEntity<Account> getAccountDetails(@RequestBody Account account1) {
```

Comment:
why is object of Account required in req?

Reply:
ok, will update this  by creating new request class.